src/ComplexTypes/BaseArrayOfType.php

```diff
@@ -14,14 +16,53 @@
     const WRAPPED_PROPERTY = '';
 
     /**
-     * Get the iterator for the current object.
+     * Implements IteratorAggregate::getIterator().
      * @return ArrayIterator
      */
     public function getIterator()
     {
+        return new ArrayIterator($this->getWrappedProperty());
+    }
+
+    /**
+     * Implements ArrayAcces::offsetExists().
+     */
+    public function offsetExists($offset)
+    {
+        return isset($this->getWrappedProperty()[$offset]);
```

Comment:
Sorry, never mind. I thought `array_key_exists()` was faster, but apparently it is the other way around.